components/textbook-beta/RelatedMaterialSection.vue

```diff
@@ -0,0 +1,103 @@
+<template>
```

Comment:
Have you checked if you can reuse the `PrerequisiteMaterialSection` component instead of creating this new `RelatedMaterialSection` component?

Reply:
I started to turn this into a reusable component but with this task being a `S`, adding refactors into this felt out of scope... so I created an issue to handle extending the component https://github.com/Qiskit/qiskit.org/issues/2167

Reply:
ok, I see your point. For this issue I'm ok with this, but I think we should reuse the components and clean code before merge the feature branch. What do you think, @techtolentino , @eddybrando , @korgan00 ?

Reply:
It is fine for me.